torch/_inductor/codegen/triton.py

```diff
@@ -474,8 +474,8 @@ def floordiv(a, b):
 
     @staticmethod
     def sign(x):
-        left = ops.where(ops.lt("0", x), 1, 0)
-        right = ops.where(ops.lt(x, "0"), 1, 0)
+        left = ops.lt("0", x)
+        right = ops.lt(x, "0")
         sub = ops.sub(left, right)
```

Comment:
Does this work? A quick check suggests `bool - bool => bool` in triton.

Reply:
I run the wrong tests locally lol. Now it should work.